scripts/components/advanced-color-picker/advanced-color-picker.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import { BaseControl, ColorPicker, __experimentalGradientPicker as GradientPicker } from '@wordpress/components';
+import { ColorPaletteCustom, CustomSelect, icons, IconLabel } from '@eightshift/frontend-libs/scripts';
+import { components } from 'react-select';
+
+const getIcon = (value) => {
+	if (!value) {
+		return icons.none;
+	}
+
+	switch (value) {
+		case 'project':
+			return icons.paletteColor;
+		case 'solid':
+			return icons.solidColor;
+		default:
+			return icons[value];
+	}
+}
+
+const CustomPickerOption = props => {
+	return (
+		<components.Option {...props}>
```

Comment:
Can these be extracted somehow? 

All those parts of React Select components are using `export default`. We could then try to import them `as ReactSelectOption` etc. just not to confuse them with our own when reading the code.